src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/SolutionExplorer_OutOfProc.cs

```diff
@@ -128,8 +128,12 @@ public void SaveAll()
         public void ShowOutputWindow()
             => _inProc.ShowOutputWindow();
 
-        public void UnloadProject(ProjectUtils.Project project)
-            => _inProc.UnloadProject(project.Name);
+        /// <summary>
+        /// Unloads the project. <paramref name="index"/> is 1 based.
+        /// </summary>
+        /// <param name="index"></param>
+        public void UnloadProject(int index)
+            => _inProc.UnloadProject(index);
```

Comment:
I'd prefer this continue to take a `ProjectUtils.Project`. Maybe have a helper that can map a `ProjectUtils.Project` to the index, and then pass that to the equivalent "in proc" API?